content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -135,30 +134,26 @@ For pods, this is any pod that has a deletion timestamp set.
 
 ### Topology information {#topology}
 
-{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
+Each endpoint within an EndpointSlice can contain relevant topology information.
+The topology information includes the location of the endpoint and information
+about the corresponding Node and zone. This will be available in the following
+per endpoint fields on EndpointSlices:
+
+* `nodeName` - The name of the Node this endpoint is on.
+* `zone` - The zone this endpoint is in.
 
 {{< note >}}
-The topology field in EndpointSlices has been deprecated and will be removed in
-a future release. A new `nodeName` field will be used instead of setting
-`kubernetes.io/hostname` in topology. It was determined that other topology
-fields covering zone and region would be better represented as EndpointSlice
-labels that would apply to all endpoints within the EndpointSlice.
+In the v1 API, the per endpoint `topology` was effectively removed in favor of
+the dedicated fields `nodeName` and `zone`.
+
+Setting arbitrary topology fields per endpoint has been deprecated and will not
+be supported in the v1 API. Instead, the v1 API supports setting individual
+node name and zone fields. These fields will be automatically translated
+between API versions. For example, the value of
+topology[topology.kubernetes.io/zone] in the v1beta1 API will be accessible in
```

Comment:
```suggestion
`topology.kubernetes.io/zone` in the v1beta1 API are available as
```

Reply:
The field in the API is `topology` which is a map. In this case we are referring to the key value pair `topology[topology.kubernetes.io/zone]`. I added backticks, does that make it clearer?

Reply:
Maybe `topology["topology.kubernetes.io/zone"]` would be slightly clearer? Agree that the `topology[` prefix adds important context here.

Reply:
I see. Since `topology` is a map, and `topology.kubernetes.io/zone` is a key in that map, we may want to revise the sentence so that the text is more clearer for YAML developers/users.

Reply:
I have called out that `topology.kubernetes.io/zone` as a key in the `topology` field in the new version. Let me know if the text is clear enough now. I am not exactly sure how to describe it for YAML users.